src/main/java/io/jenkins/plugins/appcenter/AppCenterRecorder.java

```diff
@@ -131,6 +139,96 @@ private boolean uploadToAppCenter(Run<?, ?> run, FilePath filePath, TaskListener
         }
     }
 
+    private String fetchReleaseNotes(Run<?, ?> build, FilePath filePath, EnvVars vars) {
+        File tempDir;
+        try {
+            tempDir = File.createTempFile("jtf", null);
```

Comment:
Do we need to be creating this file handler for _all_ the release notes methods? As far as I can tell it is only for the file release method. The others don't need it Correct?